src/parser/jobs/drk/modules/BloodWeapon.tsx

```diff
@@ -76,8 +78,12 @@ export default class BloodWeapon extends Module {
 		if (actionId === ACTIONS.BLOOD_WEAPON.id) {
 			const bloodWeapon = new BloodWeaponState(event.timestamp)
 			const fightTimeRemaining = this.parser.fight.end_time - event.timestamp
-			bloodWeapon.isRushing = BLOOD_WEAPON_DURATION >= fightTimeRemaining
-
+			if ( BLOOD_WEAPON_DURATION >= fightTimeRemaining ) {
+				// Rushing - end of fight, reduce expected number of skills
+				const gcdEstimate = this.globalcooldown.getEstimate(true)
```

Comment:
uberwithanumlautnit; don't need to pass `true` to `getEstimate`, it's the default